data_structures/src/chain.rs

```diff
@@ -992,6 +992,7 @@ impl Into<ExtendedSK> for ExtendedSecretKey {
 #[protobuf_convert(pb = "witnet::Hash")]
 pub enum Hash {
     /// SHA-256 Hash
+    #[allow(clippy::upper_case_acronyms)]
```

Comment:
Maybe it's better to leave `-A clippy::upper-case-acronyms` instead of adding this allow clippy everywhere. Later we can decide if we want to fix it or not.